docs/examples/driver_examples/Qcodes example with Basel SP983c Preamp.ipynb

```diff
@@ -0,0 +1,314 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "domestic-health",
+   "metadata": {},
+   "source": [
+    "# Qcodes example with Basel SP983c Preamp"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "assigned-cartridge",
+   "metadata": {},
+   "source": [
+    "This notebook shows how to setup an input offset voltage source for the Basel SP983c Preamp. The range of input offset voltage is -10 to 10 Volts. This input offset voltage is used for offsetting the voltage by the preamp in range -0.1 to 1 Volts. Let's begin!"
```

Comment:
I think the notebook should be extended to not only explain this but how the preamp works in general.

Reply:
Ok, will update it.